keyboards/flx/virgo/rules.mk

```diff
@@ -14,14 +15,14 @@ BOOTLOADER = atmel-dfu
 #   change yes to no to disable
 #
 BOOTMAGIC_ENABLE = lite      # Virtual DIP switch configuration
-MOUSEKEY_ENABLE = yes        # Mouse keys
-EXTRAKEY_ENABLE = yes        # Audio control and System control
-CONSOLE_ENABLE = yes         # Console for debug
-COMMAND_ENABLE = yes         # Commands for debug and configuration
+MOUSEKEY_ENABLE = no        # Mouse keys
```

Comment:
If you are disabling this for VIA support, these need to be in the rules.mk of the via keymap